### DISCUSSION TITLE
[Feature Request] Memory Inspector - Allow saving of custom memory regions

### DISCUSSION THREAD

**Discussion by BoomEX-UK** [RetroAchievements/RAIntegration#1015] [Ideas]
Currently in the memory inspector we can enter a memory region to search. This has been incredibly useful when knowing where some data sits but not the exact location (such as blocks of save data, blocks of level data).

It would be great if we could click Add / Save or something similar to remember this region an allow it to be chosen from the drop down that already exists

**Comment by Jamiras** [comment]
similar to #906

**Comment by Jamiras** [comment]
implemented in #1256
